slu/slu/src/api/endpoints.py

```diff
@@ -37,7 +37,12 @@ def slu(lang: str, project_name: str):
 
     Produces a json response containing intents and entities.
     """
-    supported_languages = list(CONFIG.get_supported_langauges().keys())
+    PROJECT_CONFIG = PROJECT_CONFIG_MAP.get(project_name, None)
+
+    if PROJECT_CONFIG is None:
+        return errors.missing_project_name(project_name), 404
+
+    supported_languages = list(PROJECT_CONFIG.get_supported_langauges().keys())
```

Comment:
Too much work? why not just

```python
if lang not in PROJECT_CONFIG.get_supported_langauges():
```

Reply:
this PR isn't making any changes to those lines. other than replacing it with `MODEL_CONFIG`

```python
    supported_languages = list(MODEL_CONFIG.get_supported_langauges().keys())

    if lang not in supported_languages:
        return errors.invalid_language(supported_languages)
```

has been there already for rejecting requests which are not adhering to the config specified `lang`.

if you want to make those changes, this is how it'd look.

```python

    if lang not in MODEL_CONFIG.get_supported_langauges().keys():
        return errors.invalid_language(MODEL_CONFIG.get_supported_langauges().keys())
```

Reply:
fixed it to 

```python
    if lang not in MODEL_CONFIG.get_supported_langauges():
        return errors.invalid_language(lang)
```

let me know if you have different thoughts.